src/video_provider_manager.cpp

```diff
@@ -24,6 +25,8 @@
 #include <libaegisub/log.h>
 #include <libaegisub/string.h>
 
+#include "wx/translation.h"
```

Comment:
This should use angled brackets (`<wx/translation.h>`).
(Also, it looks like most Aegisub source files include `<wx/intl.h>` instead of this, which then in turn includes `translation.h`. I don't see any reason why they'd need, to, though - if this compiles it's probably fine.)

Reply:
Indeed, the CI fails now. I'll investigate.